<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAccountWidget</class>
 <widget class="QWidget" name="CreateAccountWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(116, 88, 80);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>17</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 192, 175);
background-color:transparent;
font: 36pt &quot;BM JUA_TTF&quot;;</string>
   </property>
   <property name="text">
    <string>Create New Account</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>82</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 192, 175);
background-color:transparent;
font: 18pt &quot;BM JUA_TTF&quot;;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>111</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(116, 88, 80);
border-style: solid;
border-width: 3px;
border-color:rgb(214, 192, 175);
border-radius: 10;
background-color: rgb(197, 165, 142);
font: 20pt &quot;BM JUA_TTF&quot;;
padding-left: 10px; padding-right: 10px;</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>197</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(116, 88, 80);
border-style: solid;
border-width: 3px;
border-color:rgb(214, 192, 175);
border-radius: 10;
background-color: rgb(197, 165, 142);
font: 20pt &quot;BM JUA_TTF&quot;;
padding-left: 10px; padding-right: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>287</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(116, 88, 80);
border-style: solid;
border-width: 3px;
border-color:rgb(214, 192, 175);
border-radius: 10;
background-color: rgb(197, 165, 142);
font: 20pt &quot;BM JUA_TTF&quot;;
padding-left: 10px; padding-right: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>377</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(116, 88, 80);
border-style: solid;
border-width: 3px;
border-color:rgb(214, 192, 175);
border-radius: 10;
background-color: rgb(197, 165, 142);
font: 20pt &quot;BM JUA_TTF&quot;;
padding-left: 10px; padding-right: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>467</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(116, 88, 80);
border-style: solid;
border-width: 3px;
border-color:rgb(214, 192, 175);
border-radius: 10;
background-color: rgb(197, 165, 142);
font: 20pt &quot;BM JUA_TTF&quot;;
padding-left: 10px; padding-right: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>167</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 192, 175);
background-color:transparent;
font: 18pt &quot;BM JUA_TTF&quot;;</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>257</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 192, 175);
background-color:transparent;
font: 18pt &quot;BM JUA_TTF&quot;;</string>
   </property>
   <property name="text">
    <string>PW</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>347</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 192, 175);
background-color:transparent;
font: 18pt &quot;BM JUA_TTF&quot;;</string>
   </property>
   <property name="text">
    <string>PHONE NUMBER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>437</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 192, 175);
background-color:transparent;
font: 18pt &quot;BM JUA_TTF&quot;;</string>
   </property>
   <property name="text">
    <string>ADDRESS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>537</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 16pt &quot;BM JUA_TTF&quot;;
	background-color: rgb(197, 165, 142);
	border: none;
	border-radius: 10px;
	
	color: rgb(116, 88, 80);
	
    border-left: 3px solid rgb(214, 192, 175);
	border-right: 3px solid rgb(214, 192, 175);
	border-bottom: 3px solid rgb(214, 192, 175);
	border-top: 3px solid rgb(214, 192, 175);

}

QPushButton:hover{
	background-color: rgb(197, 165, 142);
	
	color: rgb(214, 192, 175);
	
    border-left: 3px solid rgb(214, 192, 175);
	border-right: 3px solid rgb(214, 192, 175);
	border-bottom: 3px solid rgb(214, 192, 175);
	border-top: 3px solid rgb(214, 192, 175);

}
QPushButton:pressed{
	color: rgb(214, 192, 175);
	background-color:  rgb(116, 88, 80);
	border-top: 3px solid rgb(214, 192, 175);
	border-left: 3px solid rgb(214, 192, 175);
	border-right: 3px solid rgb(214, 192, 175);
	border-bottom: 3px solid rgb(214, 192, 175);

}</string>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>537</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 16pt &quot;BM JUA_TTF&quot;;
	background-color: rgb(197, 165, 142);
	border: none;
	border-radius: 10px;
	
	color: rgb(116, 88, 80);
	
    border-left: 3px solid rgb(214, 192, 175);
	border-right: 3px solid rgb(214, 192, 175);
	border-bottom: 3px solid rgb(214, 192, 175);
	border-top: 3px solid rgb(214, 192, 175);

}

QPushButton:hover{
	background-color: rgb(197, 165, 142);
	
	color: rgb(214, 192, 175);
	
    border-left: 3px solid rgb(214, 192, 175);
	border-right: 3px solid rgb(214, 192, 175);
	border-bottom: 3px solid rgb(214, 192, 175);
	border-top: 3px solid rgb(214, 192, 175);

}
QPushButton:pressed{
	color: rgb(214, 192, 175);
	background-color:  rgb(116, 88, 80);
	border-top: 3px solid rgb(214, 192, 175);
	border-left: 3px solid rgb(214, 192, 175);
	border-right: 3px solid rgb(214, 192, 175);
	border-bottom: 3px solid rgb(214, 192, 175);

}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>180</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BM JUA_TTF</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 13pt &quot;BM JUA_TTF&quot;;
	background-color: rgb(197, 165, 142);
	border: none;
	border-radius: 10px;
	
	color: rgb(116, 88, 80);
	
    border-left: 3px solid rgb(214, 192, 175);
	border-right: 3px solid rgb(214, 192, 175);
	border-bottom: 3px solid rgb(214, 192, 175);
	border-top: 3px solid rgb(214, 192, 175);

}

QPushButton:hover{
	background-color: rgb(197, 165, 142);
	
	color: rgb(214, 192, 175);
	
    border-left: 3px solid rgb(214, 192, 175);
	border-right: 3px solid rgb(214, 192, 175);
	border-bottom: 3px solid rgb(214, 192, 175);
	border-top: 3px solid rgb(214, 192, 175);

}
QPushButton:pressed{
	color: rgb(214, 192, 175);
	background-color:  rgb(116, 88, 80);
	border-top: 3px solid rgb(214, 192, 175);
	border-left: 3px solid rgb(214, 192, 175);
	border-right: 3px solid rgb(214, 192, 175);
	border-bottom: 3px solid rgb(214, 192, 175);

}</string>
   </property>
   <property name="text">
    <string>NEW CHAT ROOM</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
